tasks/test.yaml

```
@@ -16,6 +16,8 @@ includes:
   - idam-layer: ../packages/identity-authorization/tasks.yaml
   - common-setup: https://raw.githubusercontent.com/defenseunicorns/uds-common/v1.20.4/tasks/setup.yaml
   - compliance: https://raw.githubusercontent.com/defenseunicorns/uds-common/v1.20.4/tasks/compliance.yaml
+  - playwright: ../test/playwright/private-pki/tasks.yaml
```

very nit: I think the tasks in this file are specific to private pki things. (not playwright as a whole)
```suggestion
  - private-pki: ../test/playwright/private-pki/tasks.yaml
```